scripts/mvftoms.py

```diff
@@ -645,11 +647,16 @@ def _cp_index(a1, a2, pol):
                 # Iterate over time in some multiple of dump average
                 ntime = utc_seconds.size
                 ntime_av = 0
+                skipped = 0
 
                 for ltime in range(0, ntime - tsize + 1, tsize):
```

Comment:
Why not just start at `dump_start * tsize` instead of 0? The `skipped` variable also seems superfluous.